ia/src/ml_agent.py

```
@@ -144,57 +151,117 @@ def define_needed_resources(client):
-    if needed_resources or not current_cell or count_players < required_players:
+    if client["level"] == 1:
+        required_players = 1
+
+    if needed_resources:
         client["incantation"] = False
         return
     
+    if count_players < required_players:
+        client["incantation"] = False
+        client["waiting_for_help"] = True
+        protocole.execute_command(client, utils.BROADCAST, "I_need_help_to_level_up_to_" + str(client["level"] + 1) + "_with_" + str(client["inventory"].get("food", 0)))
```

Pylint (3.3.7)

[convention] Line too long (169/100)

(C0301)